ProgramowanieBot/Handlers/MessageHandler.cs

```diff
@@ -98,10 +99,7 @@ Task HandleReminderAsync()
                     return Task.CompletedTask;
 
                 var content = message.Content;
-                if (Config.PostResolveReminderKeywords.Any(k => content.Contains(k, StringComparison.InvariantCultureIgnoreCase)))
-                    return SendReminderMessageAsync();
-
-                return Task.CompletedTask;
+                return Config.PostResolveReminderKeywords.Any(k => content.Contains(k, StringComparison.InvariantCultureIgnoreCase)) ? SendReminderMessageAsync() : Task.CompletedTask;
```

Comment:
I decided to include this small change. Forgot to include this change in the commit and PR description. Apologies.

Reply:
https://github.com/Programowanie-Discord/ProgramowanieBot/pull/5#discussion_r1186711588